src/components/TokenSelector.tsx

```diff
@@ -331,17 +366,33 @@ const TokenSelector: React.FC<Props> = ({ isDisabled, tokens, selected, onChange
   // When the search input is focused, force menu to remain open
   const onMenuInputFocus = useCallback(() => setIsFocused(true), [])
 
-  const onKeyDown = useCallback((e): void => {
-    if (e.key === 'Escape') {
-      // Close menu on `Escape`
-      e.stopPropagation()
-      setIsFocused(false)
-    } else if (!e.target.value && e.key === ' ') {
-      // Prevent a space when input in empty.
-      // That closes the menu. (/shrug)
-      e.preventDefault()
-    }
-  }, [])
+  const onKeyDown = useCallback(
+    (e: React.KeyboardEvent<HTMLInputElement> & { target: HTMLInputElement }): void => {
+      if (e.key === 'Escape') {
+        // Close menu on `Escape`
+        e.stopPropagation()
+        setIsFocused(false)
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      } else if (!e.target.value! && e.key === ' ') {
```

Comment:
Why the `!` on `.value!`?

Reply:
forces TS to accept it as not `falsy`

Reply:
TS complained at some point, but not anymore, removing